### ISSUE TITLE
It doesn't respect plasma panel's reserved space.

### ISSUE THREAD

**Issue by ralsina** [Jazqa/kwin-quarter-tiling#88]
Looks like tiled windows go under the panel. In this screenshot you can see how the bottom row in mc's terminal is hidden.

![image](https://user-images.githubusercontent.com/1579/79402908-ab42e800-7f63-11ea-9381-3c49afa7de2b.png)

Kwin 5.18.4 in Arch with git version of kwin-quarter-tiling from AUR as of today

**Comment by Jazqa** [comment]
Thanks for reporting the issue! I'll look into it this evening. Do you have any panel options enabled, that might help me reproduce it (e.g. auto-hide)?

**Comment by ralsina** [comment]
No, just the regular panel, probably what comes by default

**Comment by Jazqa** [comment]
If you right click the panel, press "edit panel" and then "more settings" on the bottom-right corner of the screen. Do you have "windows can cover" or "windows go below" enabled?

**Comment by ralsina** [comment]
Always visible

**Comment by Jazqa** [comment]
Thanks for the quick response. "Always visible" is exactly what it should be. I'll keep trying to replicate this.

**Comment by Jazqa** [comment]
Ok, latest commit was a shot in the dark as I was not able to reproduce the bug, but please tell me if updating the script fixes the bug (afaik AUR should direct to master of the git branch, so an update should be available).

**Comment by ralsina** [comment]
I switched to another script (sorry) but I'll try it later today

**Comment by Jazqa** [comment]
No worries. I'll leave the issue open in case someone else is experiencing this.

**Comment by TR4SPY** [comment]
Hello there! Nice script you got here, I am using it for few months now and enjoying it!
Sadly I have got such an issue as described here :/ I have one KDE plasma5 dock on top then I have three latte docks on the bottom and seems like the script is doing what it wants after some "moving around" or after sometime (specially after hibernation). I need to hit WIN+T to bring it back to normal. Also for me some windows don't respect the script and open on whole screen, some respect it, some do strange things... all of it is happening totally random. I have different scripts but it is happening with that script only enabled :/ I am not sure whether it is a plasma not handling the situation properly or what is going on, later I will record a video and upload it for a reference.

@Edit
I have got the video of strange behaviour literally from now, I cannot catch it doing that thing with not respecting panel's reserved space... although believe me it sometimes does it.

https://www.youtube.com/watch?v=1ksm5o4eVHg&feature=youtu.be
